### ISSUE TITLE
Enhancement

### ISSUE THREAD

**Issue by Fanie-Visagie** [kcheng1021/GaussianPro#56]
Would be great if you could incorporate advances from MIP-Splatting....

That will take this to the next level...

**Comment by kcheng1021** [comment]
Yeah! Mip-Splatting is such a great job. I would like to enhance it when I get time. Currently our project is also under some other enhancements. We are welcome to any pull requests!
